pkg/cluster/base.go

```diff
@@ -162,6 +162,7 @@ type GetClusterStatusResponse struct {
 	Location      string                     `json:"location"`
 	Cloud         string                     `json:"cloud"`
 	Distribution  string                     `json:"distribution"`
+	Spot          bool                       `json:"spot"`
```

Comment:
Shouldn't this have `omitempty` annotation since this field is optional?

Reply:
We can add it, but `bool` will always have `false` as the default value in go, so it will never really be empty.

Reply:
I've added it just in case.